build.py

```diff
@@ -208,99 +163,98 @@ def build_variable_fonts(designspace, *steps):
         )
 
     if args.static_fonts == True:
+        # Prepare masters for interpolation.
+        generator = fontmake.instantiator.Instantiator.from_designspace(designspace)
+
         for instance_descriptor in designspace.instances:
+            # Generate instances once.
+            instance = generator.generate_instance(instance_descriptor)
+            glyphsLib.interpolation.apply_instance_data_to_ufo(instance, instance_descriptor, designspace)
 
             build_font_instance(
-                generator, 
-                instance_descriptor,
+                copy.deepcopy(instance),
```

Comment:
deep copy! of course! this is why I'm not a good Python developer!

Reply:
Sooooo, I had tried using deepcopy before when setting up this build script and was running into some super irritating issues where the copy wasn't working correctly. I can't recall exactly what the error was, but it took me a week or so of banging my head against the wall to decide to just repeat the generation. I'll check tomorrow if the new build script works without that error (maybe it was due to some other library?) before I feel confident to sign off on this.